### ISSUE TITLE
Beautifier issue

### ISSUE THREAD

**Issue by zvit** [mark-wiemer/vscode-autohotkey-plus-plus#124]
### Unformatted input snippet

```autohotkey
:oc:]-::
txt := "
(
;==================================================
;// NOTE
;=================================================="
)
SendInput, %txt%
return
```

### Expected formatted output snippet

```autohotkey
;==================================================
;// NOTE
;==================================================
```

### Script after saving the ahk file
```autohotkey
:oc:]-::
    txt := "
   (
	;==================================================
	;// NOTE
	;=================================================="
    )
   SendInput, %txt%
return
```

### Actual formatted output snippet

```autohotkey
	;==================================================
		;// NOTE
			;==================================================
```

### Additional context
Yes, I can just save the file and the generated NOTE will get pushed to the side correctly, but I have my VS Code set so that it auto saves while typing so that I don't have to press CTRL-save all the time. This is an issue for me.

**Comment by mark-wiemer** [comment]
Hi @zvit, can you use code fencing (3 back ticks) to show the code before and after formatting? See [Formatter bug template](https://github.com/mark-wiemer/vscode-autohotkey-plus-plus/issues/new?assignees=mark-wiemer&labels=bug%2C+formatter&template=formatting-bug.md&title=)

**Comment by zvit** [comment]
Done. Thank you.

**Comment by mark-wiemer** [comment]
Thanks @zvit , but can I ask for your help once again? I'm still a bit confused: What code are you trying to format, and what does it look like? Can you try providing just two large code blocks instead of 4 smaller ones? In short, I'm not able to reproduce the behavior you've mentioned.

Here is what I get:

Unformatted input snippet:

```ahk
:oc:]-::
txt := "
(
;==================================================
;// NOTE
;=================================================="
)
SendInput, %txt%
return
```

Actual formatted output snippet:

```
:oc:]-::
    txt := "
    (
        ;==================================================
        ;// NOTE
        ;=================================================="
    )
    SendInput, %txt%
return
```

Expected formatted output snippet (my best guess):

```
:oc:]-::
    txt := "
    (
;==================================================
;// NOTE
;=================================================="
    )
    SendInput, %txt%
return
```

Does this seem right?

**Comment by fade2gray** [comment]
Too many backticks ;)

**Comment by zvit** [comment]
I think I was confused as to what "Output Snippet" meant. My assumption was that it means the generated text after running the script and pressing the hotstring. However, if it means what I now think it does--the visual script itself after saving via CTRL-S--then your last reply is correct.

The issue is not the AKH++ beautifier itself (visually, it makes the code easier to read), but rather the resulting output caused by the beautifier when the code is running and the hotstring is pressed,

Note: When the hotstring is pressed in a different program, e.g., notepad, word, chrome, etc., only the tabs are visible, as so:

```autohotkey
      ;==================================================
      ;// NOTE
      ;==================================================
```
But in VS Code, for some reason, it also cascades the text:
```autohotkey
    ;==================================================
        ;// NOTE
            ;==================================================
```
Bottom line, when my script is running and I press the hotstring in VS Code, I want the result to be that the text is justified to the left, like this:
```autohotkey
;==================================================
;// NOTE
;==================================================
```
I hope this is more clear.

**Comment by fade2gray** [comment]
Try
```ahk
:oc:]-::
txt := "
( LTrim Join`n
;==================================================
;// NOTE
;==================================================`n
)"
SendInput, %txt%
return
```

Just an observation: according to the docs (line continuation/method 2/example 2), the closing quote should come after the closing parenthesis.

**Comment by zvit** [comment]
> Try
> 
> ```ahk
> :oc:]-::
> txt := "
> ( LTrim Join`n
> ;==================================================
> ;// NOTE
> ;==================================================`n
> )"
> SendInput, %txt%
> return
> ```
> 
> Just an observation: according to the docs (line continuation/method 2/example 2), the closing quote should come after the closing parenthesis.

That worked!! Thanks!
